### PR TITLE
Bump build version to rebuild for new ittapi and new gcc runtime

### PR DESCRIPTION
**Pull request by oleksandr-pavlyk**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by oleksandr-pavlyk** [comment]
@conda-forge-admin, please rerender

**Comment by oleksandr-pavlyk** [comment]
@conda-forge-admin @vnlitvinov This feedstock went unmaintained for some time. Is there a way to merge this PR without involvement of the currently listed maintainer? The PR sets myself as a maintainer so this won't reoccur.

File: recipe/meta.yaml
Comment (vnlitvinov) [inline recipe/meta.yaml:9]: Is there a reason to use a personal repo? I would propose to create an organization or (IMHO, a better approach) to move it under some Intel-owned org.

**Comment by vnlitvinov** [comment]
> @conda-forge-admin @vnlitvinov This feedstock went unmaintained for some time. Is there a way to merge this PR without involvement of the currently listed maintainer? The PR sets myself as a maintainer so this won't reoccur.

I can approve and merge it, so when you're set as the maintainer (after this PR is merged) you will be able to push the feedstock forward as you see fit.

P.S. In case something like this happens next time, I believe there are some support people in conda-forge organization (IIRC you have to contact `@conda-forge/core`) who can help with ownership transfer when given enough evidence (but `@conda-forge-admin` is a bot, so it cannot help).

File: recipe/meta.yaml
Comment (oleksandr-pavlyk) [reply recipe/meta.yaml:9]: The `itt-python` is not an official Intel-authored package. I forked it per agreement with @rcthomas to maintain it without the need to involve him in managing the original repo NERSC/itt-python.

Intel may choose to develop its own Python API to ITT and publish it with [ittapi](https://github.com/intel/ittapi) repo the same way the Rust API is published there now.

Comment (vnlitvinov) [reply recipe/meta.yaml:9]: Okay, you're in the driver seat now then! 🚗

**Review by vnlitvinov — APPROVED** [review]

**Pull request merged**
